src/main/java/duke/Scripts.java

```diff
@@ -0,0 +1,17 @@
+package duke;
+
+public class Scripts {
+    String greeting = "Hello! I'm LimBay.\nWhat can I do for you?\n";
+    String bye = "Bye. Hope to see you again soon!";
+
+    public Scripts() {
```

Comment:
Should this have been deleted? Or did you intend to declare the constructor without any implementation? Because this code is a bit redundant hence it's best to discard it